### DISCUSSION TITLE
Google Search Console clicks are much higher than Plausible visits

### DISCUSSION THREAD

**Discussion by DemiliciousOne** [plausible/analytics#4835] [Self-Hosted Support]
GSC clicks have always been over double the Total Visits shown on Plausible. What could be causing this?

GSC
![Screenshot 2024-11-18 at 5 06 52 PM](https://github.com/user-attachments/assets/9cf3167b-d7b8-4891-8181-c9203692d49f)

Plausible
![Screenshot 2024-11-18 at 5 06 44 PM](https://github.com/user-attachments/assets/6b5c445b-c905-485c-b129-c2e63edc7f73)

**Comment by ruslandoga** [comment]
👋 @DemiliciousOne 

I think clicks from a search results page are more like pageviews and not visits (sessions). ChatGPT seems to agree: https://chatgpt.com/share/673b71c0-aa94-8010-aff6-ca9c32ba22cf

**Reply by ruslandoga** [reply]
And from that viewpoint the data seems consistent as pageviews from Google (clicks) would be a subset of all pageviews. But I am not on the core team and don't really know much about the Google stuff so I might be wrong.
